package.json

```diff
@@ -44,18 +44,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "go-ipfs-dep": "0.4.1",
+    "bl": "^1.1.2",
+    "go-ipfs-dep": "0.4.3",
```

Comment:
You need to update to `0.4.3-1`, `0.4.3` has a bug that is only detected when it gets installed after publish (impossible to detect beforehand)